io.catenax.fleet.vehicles/RELEASE_NOTES.md

```diff
@@ -1,8 +1,17 @@
 # Changelog
 All notable changes to this model will be documented in this file.
 
-## [Purpose of this model]
-This aspect model was created in the Catena-X use case quality. The purpose of this aspect model is to transfer general information of a list/fleet of vehicles - primarily how the vehicles are equipped, when they were built and which engines are installed.
+## [Unreleased]
+- Version 2.0.0 of model is deprecated. It has a wrong regex for BPNS validation from the old shared model for BPNS trait.
+
+## [2.1.0] - 2024-02-12
```

Comment:
please change the date

Reply:
I'm not able to do that due to missing rights on this branch